### PR TITLE
Newtests/math a

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: test/test_distribution.hpp [hunk 1]
```diff
@@ -69,7 +69,7 @@ namespace eve
       if(aa == bb) res = aa;
       else if(bb <= aa+half(as(aa))) res = aa+(bb-aa)*sd(gen);
       else if((aa >= 0 && bb <= 1) || (bb <= 0 && aa >= -1)) res = aa+(bb-aa)*sd(gen);
-      else if(aa >= -1 && bb <= 1)  res =((sd(gen)*(bb-aa) > -aa) ? -aa : bb)*sd(gen);
+      else if(aa >= -1 && bb <= 1) res =((sd(gen)*(bb-aa) > -aa) ? aa : bb)*sd(gen);
```
Comment (jfalcou) [inline test/test_distribution.hpp:72]: why the change ?

Comment (jtlap) [reply test/test_distribution.hpp:72]: because it was a mistake
